org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/XMLAssert.java

```diff
@@ -128,6 +129,11 @@ public void collectDocumentToValidate(Predicate<DOMDocument> validateDocumentPre
 
 		}
 
+		@Override

```

Comment:
Is it required since the interface defines a default implementation?

Reply:
Fixed